packages/api/src/controllers/asset.ts

```diff
@@ -263,6 +263,13 @@ function getDownloadUrl(
   asset: WithID<Asset>,
   os: ObjectStore
 ): string {
+  // If mp4 rendition file exists, then use it as download url
+  const staticPlaybackInfos = getStaticPlaybackInfo(asset, os);
+  if (staticPlaybackInfos.length > 0) {
+    return staticPlaybackInfos[0].playbackUrl;
```

Comment:
I don't think we have a guarantee that this array will be ordered. This should probably pick the rendition with the highest quality instead of just returning the first one.

Reply:
They are sorted by bitrate, so it should be ok. Plus, we'll generate only one MP4 rendition for recordings. [Here's](https://linear.app/livepeer/issue/VID-127/catalyst-api-generate-mp4-for-only-highest-quality-rendition) the related ticket.